src/qtfirebaseadmob.cpp

```diff
@@ -486,6 +487,26 @@ void QtFirebaseAdMobBase::setAdUnitId(const QString &adUnitId) {
     }
 }
 
+void QtFirebaseAdMobBase::setVisible(bool visible)
+{
+    if(!_ready) {
+        qDebug() << this << "::setVisible native part not ready";
+        return;
+    }
+
+    if(!_loaded) {
+        qDebug() << this << "::setVisible native part not loaded - so not changing visiblity to" << visible;
+        return;
+    }
+
+    if(!_visible && visible) {
+        show(); // NOTE show will change _visible and emit signal
+    } else {
+        // NOTE An interstitial can't be hidden by any other than the user
```

Comment:
`// NOTE An interstitial can't be hidden by any other than the user`
How do you check if this is an interstitial when all this is moved to the Base class?